vocabulary-draft/introduction.md

```diff
@@ -0,0 +1,26 @@
+# Continuous Delivery Events Vocabulary
+
+__Note:__ This is a work-in-progress draft version and is being worked on by members of the Events SIG. You are very welcome to join the work and the discussions!
+
+This document intends to describe a set of events related to different phases of a Continuous Delivery process. 
+These events are categorized by meaning and the phase where they are intenteded to be used. 
+These events are agnostic from any specific tool and are designed to fit a wide range of scenarios. 
+
+The phases covered by this proposal are:
+
+- **Source Code Version Control Events**: Events emitted by changes in source code or by the creation, modification or deletion of new repositories that hold source code.
+- **Continuous Integration Pipelines Events**: includes events related to building, testings, packaging and releasing software artifacts, usually binaries.
+- **Continuous Deployment Pipelines Events**: include events related with environments where the artifacts produced by the integration pipelines actually run. These are usually services running in a specific environment (dev, QA, production), or embedded software running in a specific hardware. 
+
+These phases can also be considered as different profiles of the vocabulary that can be adopted independently. 
```

Comment:
That's an interesting idea, we'll have to define clearly in the spec to be compliant. 
I think for each event group we can say which information is require, but we cannot force platform to provide events about areas they do not cover.

Reply:
@afrittoli I think that we need to be super clear about that exactly. We should state something like:
```
As an adopter, you can choose which events fit for you and just adopt those. There is no need for every adopter to emit or consume all these events. 
```